[2025-01-08T07:41:09.228+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-08T07:41:09.290+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-01-08T07:40:03.706833+00:00 [queued]>
[2025-01-08T07:41:09.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-01-08T07:40:03.706833+00:00 [queued]>
[2025-01-08T07:41:09.308+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-01-08T07:41:09.334+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_payments> on 2025-01-08 07:40:03.706833+00:00
[2025-01-08T07:41:09.344+0000] {standard_task_runner.py:63} INFO - Started process 3093 to run task
[2025-01-08T07:41:09.350+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'transform.transform_dim_payments', 'manual__2025-01-08T07:40:03.706833+00:00', '--job-id', '907', '--raw', '--subdir', 'DAGS_FOLDER/e_commerce_dw_dag.py', '--cfg-path', '/tmp/tmppaum4_sf']
[2025-01-08T07:41:09.355+0000] {standard_task_runner.py:91} INFO - Job 907: Subtask transform.transform_dim_payments
[2025-01-08T07:41:09.469+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.transform.transform_dim_payments manual__2025-01-08T07:40:03.706833+00:00 [running]> on host 4ff5b0867c78
[2025-01-08T07:41:09.636+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='transform.transform_dim_payments' AIRFLOW_CTX_EXECUTION_DATE='2025-01-08T07:40:03.706833+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-08T07:40:03.706833+00:00'
[2025-01-08T07:41:09.641+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-08T07:41:09.684+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-01-08T07:41:09.715+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-01-08T07:41:11.545+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-01-08T07:41:11.898+0000] {logging_mixin.py:188} INFO - Đã transform và lưu dữ liệu vào dim_payments
[2025-01-08T07:41:11.900+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-01-08T07:41:11.901+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-08T07:41:11.921+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=transform.transform_dim_payments, run_id=manual__2025-01-08T07:40:03.706833+00:00, execution_date=20250108T074003, start_date=20250108T074109, end_date=20250108T074111
[2025-01-08T07:41:11.984+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-08T07:41:12.064+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-08T07:41:12.074+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
